src/components/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useItem } from '../context/ItemContext';
+
+export default function Header() {
+  const { items, handleReset } = useItem();
+
+  const handleSubmit = () => {
```

Comment:
I might name this differently. `handleSubmit` suggests this function will be called in an `onSubmit`